src/Nancy.Testing/BrowserResponseBodyWrapper.cs

```diff
@@ -13,24 +14,35 @@ public class BrowserResponseBodyWrapper : IEnumerable<byte>
         private readonly string contentType;
         private DocumentWrapper responseDocument;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BrowserResponseBodyWrapper"/> class.
+        /// </summary>
+        /// <param name="response">The <see cref="Response"/> to wrap.</param>
         public BrowserResponseBodyWrapper(Response response)
         {
             var contentStream = GetContentStream(response);
 
-            this.responseBytes = contentStream.ToArray();
+            this.responseBytes = ((MemoryStream)contentStream.BaseStream).ToArray();
             this.contentType = response.ContentType;
         }
 
-        internal string ContentType
+        /// <summary>
+        /// Gets the content type of the wrapped response.
+        /// </summary>
+        /// <returns>A string containing the content type.</returns>
+        public string ContentType
         {
             get { return this.contentType; }
         }
 
-        private static MemoryStream GetContentStream(Response response)
+        private static UnclosableStreamWrapper GetContentStream(Response response)

```

Comment:
Urg.. yes.. for some reason I got it in my head that it was being used elsewhere as well.. will fix